src/api/types/rest.ts

```diff
@@ -473,10 +473,11 @@ export interface Collection extends CollectionBase {
     data: number[];
 }
 
-export function isCollection(object: {}): object is Collection {
-    return _.all(['description', 'settings', 'data'], (property) =>
-        object.hasOwnProperty(property)
-    );
+export function isCollection(object: Collection | SampleBase | Data): object is Collection {
```

Comment:
I think `Collection` should have been `CollectionBase`. This doesn't compile with inputs that can be `CollectionHydrateData`

Although the `object.hasOwnProperty('data')` condition is a bit strange then